zerver/lib/events.py

```diff
@@ -1652,7 +1653,7 @@ def do_events_register(
                 linkifier_url_template=linkifier_url_template,
                 user_list_incomplete=user_list_incomplete,
             )
-        except RestartEventError:
+        except ReloadWebClientError:
             # This represents a rare race condition, where Tornado
             # restarted (and sent `restart` events) while we were waiting
             # for fetch_initial_state_data to return. To avoid the client
```

Comment:
I'm not entirely sure if this code path still makes sense following this PR.

Reply:
Yeah, as the window between just-before-restart and getting the `reload-web-client` event widens to longer than `/` is expected to take to load, it becomes probably-impossible to trigger.

Reply:
Added a commit to remove it.